src/Payment.res

```diff
@@ -16,6 +16,7 @@ let make = (~paymentMode, ~integrateError, ~logger) => {
   let showFields = Recoil.useRecoilValueFromAtom(showCardFieldsAtom)
   let selectedOption = Recoil.useRecoilValueFromAtom(selectedOptionAtom)
   let paymentToken = Recoil.useRecoilValueFromAtom(paymentTokenAtom)
+  let paymentMethodList = Recoil.useRecoilValueFromAtom(paymentMethodList)
```

Comment:
We have another recoil, which contains value inside of Loaded enum called paymentMethodListValue. Could you please check if that suits ur use case better